indsl/ts_utils/operators.py

```diff
@@ -235,3 +235,36 @@ def arithmetic_mean_many(data: List[Union[pd.Series, float]], align_timesteps: b
     for new_ts in data:
         timeseries_sum = add(timeseries_sum, new_ts, align_timesteps)
     return op.truediv(timeseries_sum, n)
+
+
+@check_types
+def average(
+    data: pd.Series,
+    threshold: Optional[float] = None,
+    condition: Literal["Above", "Below", "No threshold"] = "No threshold",
+) -> pd.Series:
+    """Average of a time series.
+
+    Plain simple average that computes the sum of the values of the observations in the currently displayed time window in Charts, and divides it by the number of observations within that time window.
+    The result is a constant time series with the same length as the input time series, where each value is equal to the average of the input time series.
```

Comment:
Would be good to specify that the timestamps are the same, the time series is not just the same length.